### DISCUSSION TITLE
GSoC: Seeking Advice + Reaching Out to Mentors of Project 11 (PyTorch Model Optimizations with torch.compile OpenVINO Backend)

### DISCUSSION THREAD

**Discussion by YutingGao7** [openvinotoolkit/openvino#23239] [Google Summer of Code]
Hi Dear GSOC Project Mentors of OpenVINO,

I'm Yuting, a master's student of computer science at San Francisco State University, specializing in ML and NLP. I'm thrilled to apply for the GSOC project 11: "PyTorch Model Optimizations with torch.compile OpenVINO Backend." 

My experience includes working with various ML models, from predictive transformer models (BERT) to generative decoder models like (Flan-t5, Llama2, and GPTs). I've tackled challenges related to optimizing neural network parallelism for CPUs and GPUs, as well as implementing model compression techniques like quantization and sparsity. I'm proficient in PyTorch frameworks for constructing, training, analyzing, and deploying ML models. I'm enthusiastic about this project to delve into OpenVINO's core functionality in supporting a variety of cutting-edge LLMs. I'm both detail-oriented and deadline-driven, and I thrive in environments where I can continuously learn and grow.

In my contributions to the OpenVINO project, I've submitted PRs at https://github.com/openvinotoolkit/openvino/pull/23212 and https://github.com/openvinotoolkit/openvino/pull/23213. The first PR addressed a build issue, and the second updated the Readme documentation. While working on the taken GFI ticket, I noticed that our Readme Doc remains outdated regarding the make methods while the Contributing Guide has been updated promptly, incurring discrepancy or confusion, so I took the initiative to update it and created a follow-up PR for peer contributors.

I'm deeply passionate about learning and contributing to this toolkit project. However, as an open-source beginner, I have limited open-source contributing experience so far. I'm wondering if this might negatively impact the evaluation of my application for the GSOC project. Are there specific aspects I could highlight in my application to compensate for this? Any advice you could offer would be greatly appreciated! 

The opportunity to join your vibrant community for a meaningful summer would be truly exciting for me. I'm looking forward to hearing from you soon!

**Comment by adrianboguszewski** [comment]
@ravi9 @suryasidd please look at it

**Reply by ravi9** [reply]
Hi @YutingGao7, Thank you for your interest and sharing your PRs.

Please email me your "application" for review. See [Application Template.](https://github.com/openvinotoolkit/openvino/wiki/Google-Summer-Of-Code#application-template)
We can provide feedback and you could submit it via GSoC portal.
Please email me with any specific questions you might have.
